docs/about/talks.md

```diff
@@ -0,0 +1,12 @@
+# Talks and workshops
+
+On this page you will find a collection of our talks and workshop videos showcasing Juvix.
+
+In these videos we delve into the fascinating world of this powerful language,
+uncovering its core principles, and demonstrating real-world examples of Juvix
+applications.
+
+Gain valuable insights and inspiration from our presentations at various conferences.
+
+- ["Juvix"](https://youtu.be/T8jkl7_BMAY) _@Privacy Evolution 22_ by **Paul Cadman**
+- ["Juvix: Towards a Functional Programming Language for Decentralized Applications and Beyond"](https://youtu.be/YAOFCQTAhUI) _@ETHPrague2023_ by **Veronika Romashkina**
```

Comment:
Put this list inside a section like `## 2023`.

Let's use a format like the one described here: https://www.scribbr.com/apa-examples/speech/ for talks.

Reply:
I used that. But kept names in the NAME SURNAME order for being less formal in a way 🙂